src/sentry/templates/sentry/create-project.html

```diff
@@ -27,7 +27,16 @@
       {{ form.team|as_crispy_field }}
     {% endif %}
 
+    {% if 'signup' in request.GET %}
```

Comment:
You don't need to do this. `?signup` will already be in the `GET` for this page, which means the `action` is going to `POST` back to this same url. Inside your view, you could just check for `'signup' in request.GET` again without this extra `POST` field.